internal/handler/handlers_test.go

```
@@ -45,38 +206,165 @@ func TestServer_LogIn(t *testing.T) {
 			defer ctrl.Finish()
 
 			auth := mock_service.NewMockAuth(ctrl)
-			tc.mock(auth, tc.inputUserLogin, tc.inputUserPassword)
-
-			s := Server{
-				Auth: auth,
-			}
+			tc.mock(auth, tc.serviceName, tc.servicePassword)
 
-			r := mux.NewRouter()
-			r.HandleFunc("/user/login", s.LogIn).Methods("POST")
+			s := NewServer(auth, nil)
 
 			w := httptest.NewRecorder()
-			req := httptest.NewRequest("POST", "/auth/login", bytes.NewBufferString(tc.inputBody))
+
+			request, _ := json.Marshal(tc.requestBody)
+			req := httptest.NewRequest("POST", "/auth/signup", bytes.NewBuffer(request))
 
 			s.Router.ServeHTTP(w, req)
 
-			assertStatusCode(t, w.Code, tc.expectedStatusCode)
-			assertBody(t, w.Body.String(), tc.expectedResponseBody)
+			if tc.isErrorExpeced {
+				var response ErrorResponse
+				_ = json.Unmarshal(w.Body.Bytes(), &response)
```

In general, ignoring errors is not good and since you use testify you can `assert.NoErr()`